src/repl/scala/tools/nsc/interpreter/ReplVals.scala

```diff
@@ -17,12 +17,11 @@ import scala.reflect.runtime.{universe => ru}
 abstract class ReplVals { }
 
 class StdReplVals(final val intp: IMain) extends ReplVals {
-  // TODO bring back repl, reader, completion and history?
+  // TODO bring back repl, reader, isettings, completion and history?
```

Comment:
please just don't call it `isettings`, if it must be recalled. I realize these names were conceived in the aughts, but one ought not `i-` everything.